assets/app/views/builds.html

```diff
@@ -143,11 +141,15 @@
             </dl>
           </div>
         </div>
-        <div class="well" style="margin-bottom: 10px;" ng-repeat="build in buildsByBuildConfig[buildConfigName] | orderObjectsByDate : true">
+        <div class="well build-log" ng-repeat="build in buildsByBuildConfig[buildConfigName] | orderObjectsByDate : true">
           <h3>{{build.metadata.name}}</h3>
-          <div>Created: <relative-timestamp timestamp="build.metadata.creationTimestamp"></relative-timestamp></div>
-          <div>
-            <span style="margin-right: 5px;">Status: {{build.status.phase}}</span>
+          <div class="row">
+            <div class="col-md-6">
+              <div class="log-meta">
+          <span class="log-meta-title">Created:</span><relative-timestamp timestamp="build.metadata.creationTimestamp"></relative-timestamp>
+              </div>
+          <div class="log-meta">
+            <span class="log-meta-title">Status:</span>
```

Comment:
Indentation is a little off here. I'm having trouble seeing the structure and what tags are supposed to be closed where.